src/Features/CSharp/Portable/QualifyMemberAccess/CSharpQualifyMemberAccessDiagnosticAnalyzer.cs

```diff
@@ -21,14 +21,20 @@ protected override bool IsAlreadyQualifiedMemberAccess(ExpressionSyntax node)
 
         // If the member is already qualified with `base.`,
         // or member is in object initialization context,
-        // or member in property or field initialization, it cannot be qualified.
+        // or member in property or field initialization,
+        // or member in base constructor,  it cannot be qualified.
         protected override bool CanMemberAccessBeQualified(ISymbol containingSymbol, SyntaxNode node)
         {
             if (node.GetAncestorOrThis<AttributeSyntax>() != null)
             {
                 return false;
             }
 
+            if (node.GetAncestorsOrThis(n => n.IsKind(SyntaxKind.BaseConstructorInitializer) && n.Contains(node)).Any())
```

Comment:
💭 Is there any reason why we can't simplify this to the following?

```csharp
if (node.GetAncestorOrThis<ConstructorInitializerSyntax>() != null)
```

Reply:
No reason other then I hadn't realized I could simplify it at the time :-)
Please see new commit which addresses this.